Source/Core/Core/HW/WiimoteCommon/WiimoteConstants.h

```diff
@@ -0,0 +1,45 @@
+// Copyright 2010 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+// Wiimote internal codes
+
+// Communication channels
+namespace WiimoteChannel
+{
+constexpr u8 const OUTPUT_CHANNEL = 0x11;
+constexpr u8 const INPUT_CHANNEL = 0x13;
+};
+
+// The 4 most significant bits of the first byte of an outgoing command must be
+// 0x50 if sending on the command channel and 0xA0 if sending on the interrupt
+// channel. On Mac and Linux we use interrupt channel; on Windows, command.
+namespace WiimoteReport
+{
+#ifdef _WIN32
+constexpr u8 const SET_REPORT = 0x50;
```

Comment:
Where should the ``#ifdef`` go then? ``SET_REPORT`` is used several times. Or do you have a different suggestion?